r2angr/angr

```diff
@@ -3,11 +3,78 @@ CWD=$(dirname $0)
 OLD=$PWD
 cd $CWD
 if [ ! -d venv ]; then
-    python -m venv venv
+    if ! command -v termux-setup-storage; then
```

Comment:
Be positive. Install termux stuff if the condition matches. Otherwise do other stuff not the other way around

Reply:
Done

Reply:
😅